kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -12153,7 +12153,13 @@ def start(self):
         http_server.register_async_uri(self.SET_VM_VF_NIC_STATE, self.set_vf_nic_state)
         http_server.register_async_uri(self.SET_VF_NIC_MAC_PATH, self.set_vf_nic_mac)
 
-        self.clean_old_sshfs_mount_points()
+        # run sshfs cleanup in background to avoid blocking the startup thread
+        # (fusermount can block for up to 2s per stale mount point)
+        @thread.AsyncThread
```

Ruff (0.15.4)

[error] `thread` may be undefined, or defined from star imports

(F405)